OpenRA.Mods.Common/OpenRA.Mods.Common.csproj

```diff
@@ -1,4 +1,4 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Please try to avoid committing changes like this and the last hunk in this file (last line)